### PR TITLE
Please review this @casey-collab

### PR DESCRIPTION
**Pull request by devakara**
Please review this @casey-collab Thank you.

### CODE REVIEW

File: telugu/prayer-telugu/prayer-telugu.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+There is no telugu story added as of yet. So, I'll just start the thread by adding a prayer.
```
Comment (casey-collab) [inline telugu/prayer-telugu/prayer-telugu.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: telugu/telangana-telugu/Telanganatelugu.md [hunk 2]
```diff
@@ -11,4 +11,6 @@
    అస్తిత్వాన్ని చాటుకోవడమే. 
    ఇదంతా కవి అంతర్గత, బాహ్య 
    వలసాధిపత్యాలను వ్యతిరేకించే 
-   క్రమంలో చేసే కవితా గానమే.
\ No newline at end of file
+   క్రమంలో చేసే కవితా గానమే.
+
+Starting with a prayer. Please click [here](telugu/prayer-telugu/prayer-telugu.md).
```
Comment (casey-collab) [inline telugu/telangana-telugu/Telanganatelugu.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by devakara** [comment]
Added my page's link to the existing story. Please review this @casey-collab Thank you.

File: telugu/prayer-telugu/prayer-telugu.md (hunk 1 above)
Comment (casey-collab) [inline telugu/prayer-telugu/prayer-telugu.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by devakara** [comment]
Updated the link to point to the correct page. Please review this @casey-collab Thank you.

**Comment by casey-collab** [comment]
Looks great! :lollipop:

**Pull request merged**